src/Apps/Settings/Routes/SavedSearchAlerts/Components/NewSavedSearchAlertEditForm.tsx

```diff
@@ -69,7 +69,7 @@ const NewSavedSearchAlertEditSteps: React.FC<NewSavedSearchAlertEditStepsProps>
     <>
       <Media greaterThanOrEqual="md">
         {current === "ALERT_DETAILS" && (
-          <Box flex={1} p={4}>
+          <Box p={4} minWidth={[null, null]}>
```

Comment:
I am not sure if this is a common pattern in web but i would have assumed the minWidth to be null anyway if not specified or 0. @olerichter00 maybe knows better here